src/plugins/recordTypes/uoc/fields.js

```diff
@@ -0,0 +1,310 @@
+import { defineMessages } from 'react-intl';
+
+export default (configContext) => {
+  const {
+    CompoundInput,
+    DateInput,
+    // OptionPickerInput,
+    TextInput,
+    AutocompleteInput,
+    // IDGeneratorInput,
+    TermPickerInput,
+  } = configContext.inputComponents;
+
+  const {
+    configKey: config,
+  } = configContext.configHelpers;
+
+  const {
+    DATA_TYPE_DATE,
+  } = configContext.dataTypes;
+
+  const {
+    extensions,
+  } = configContext.config;
+
+  return {
+    document: {
+      [config]: {
+        view: {
+          type: CompoundInput,
+          props: {
+            defaultChildSubpath: 'ns2:uoc_common',
+          },
+        },
+      },
+      ...extensions.core.fields,
+      'ns2:uoc_common': {
+        [config]: {
+          service: {
+            ns: 'http://collectionspace.org/services/uoc',
+          },
+        },
+        referenceNumber: {
+          [config]: {
+            cloneable: false,
+            messages: defineMessages({
+              name: {
+                id: 'field.uoc_common.referenceNumber.name',
+                defaultMessage: 'Reference number',
+              },
+            }),
+            required: true,
+            view: {
+              type: TextInput,
+            },
+          },
+        },
+        methodList: {
```

Comment:
The schema generation makes an inner complex type for fields ending with "List", right? And there's some app layer config property to make it not do that? Just making sure you did that.

Reply:
Looks like for multi-valued fields with scalar data (like this field), that App layer config attribute doesn't matter.  Nuxeo is smart enough to do the right thing when generating the DB tables.